.github_changelog_generator

```diff
@@ -1,4 +1,4 @@
-future-release=v0.31.0
+future-release=v0.32.0
```

Comment:
can you please check if this should be 0.33.0?
Maybe this is why you had this issue: `I followed the official release process but got some strange results. with duplicated version`